src/components/RecommendActivity/EventCard.tsx

```diff
@@ -0,0 +1,187 @@
+'use client';
+
+import { Skeleton } from '@components/ui/skeleton';
+import cn from '@lib/utils';
+import { cva } from 'class-variance-authority';
+import {
+  Bookmark,
+  Building2,
+  CalendarDays,
+  Check,
+  MapPin,
+  Users,
+} from 'lucide-react';
+import Image from 'next/image';
+import { useState } from 'react';
+import { FormatDate } from './RecommendActivity-types';
+
+interface EventCardProps {
+  title: string;
+  cover: string;
+  description: string;
+  startTime: FormatDate<'YY.MM.DD'>;
+  endTime: string;
+  attendeeCount: number;
+  isCollected: boolean;
+  location: string;
+  organizer: string;
+  pricing: number;
+  discount: number | undefined;
+}
+
+const EventCard = ({
+  title,
+  cover,
+  description,
+  startTime,
+  endTime,
+  attendeeCount,
+  isCollected = false,
+  location,
+  organizer,
+  pricing,
+  discount,
+}: EventCardProps) => {
+  const [collected, setCollected] = useState(isCollected);
+
+  const collectedVariants = cva(
+    'absolute bottom-0 right-0 flex h-20 w-20 flex-col items-center justify-center gap-2 text-xs transition ease-out duration-300',
```

Comment:
增加 font-wight 和 line-height
```suggestion
    'absolute bottom-0 right-0 flex h-20 w-20 flex-col items-center justify-center gap-2 text-xs transition ease-out duration-300 font-medium leading-5',
```